src/test_workflow/benchmark_test/benchmark_test_cluster.py

```diff
@@ -157,7 +155,7 @@ def setup_cdk_params(self, config: dict) -> dict:
             "region": config["Constants"]["Region"],
             "suffix": suffix,
             "securityDisabled": str(self.args.insecure).lower(),
-            "adminPassword": 'myStrongPassword123!' if need_strong_password else None,
+            "adminPassword": 'myStrongPassword123!' if password else None,
             "cpuArch": self.manifest.build.architecture if self.manifest else 'x64',
```

Comment:
We should use the util `get_password` here, @peterzhuamazon WDYT?

Reply:
I was also confused about this, but it seems like this specific test is trying to set "myStrongPassword123!" or None depending on the version, instead of "myStrongPassword123!" or "admin". Thus, I had to make the usage of the util function in this way.

Reply:
@derek-ho you can simply omit this [block](https://github.com/opensearch-project/opensearch-build/blob/6cb96d5a3c28d07117952b66ded7f2c318fb342f/src/test_workflow/benchmark_test/benchmark_test_cluster.py#L115C9-L122C50) and just add a new line
password = get_password(self.args.distribution_version). This will return the desired value for given version and you can eliminate using semver module in the files. Add similar approach [here](https://github.com/opensearch-project/opensearch-build/blob/6cb96d5a3c28d07117952b66ded7f2c318fb342f/src/test_workflow/benchmark_test/benchmark_test_cluster.py#L146C1-L151C44)
https://github.com/opensearch-project/opensearch-build/blob/6cb96d5a3c28d07117952b66ded7f2c318fb342f/src/test_workflow/benchmark_test/benchmark_test_cluster.py#L160

Reply:
Feel free to move the location of utils.py as it is used even outside test_workflow